DuckDuckGo/TabBar/View/TabBarViewController.swift

```diff
@@ -71,6 +71,8 @@ final class TabBarViewController: NSViewController, TabBarRemoteMessagePresentin
     private let pinnedTabsViewModel: PinnedTabsViewModel?
     private let pinnedTabsView: PinnedTabsView?
     private let pinnedTabsHostingView: PinnedTabsHostingView?
+
+    var disableTabPreviews: Bool = false
```

Comment:
The name may be confusing to readers (I needed a moment myself :D). Could we rename it as `shouldDisplayTabPreviews`? That would also reverse the logic so you'd need a `guard else` instead of `if` down in this class.